app/views/groups/create/groupType/exceed_group_selection.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import controllers.routes
+@import views.html.main_layout
+
+@this(
+        layout: main_layout,
+        p: p, h1: h1,
+        button: link_as_button
+)
+
+
+
+@(
+
+        )(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
+
+@layout(
+
+    title = msgs("All access groups have been created"), // check if wording is ok
+    backLinkHref = Some(routes.CreateGroupController.showConfirmGroupName.url)){
+
+    @h1(msgs("access-group-name-exceed-groups.title")) @*Add welsh*@
+
+    @p(msgs("access-group-name-exceed-groups.p"))@*Add welsh*@
+
+    <div>
+    @button(key = msgs("group.existing.custom.groups.button"), href = routes.CreateGroupSelectGroupTypeController.submitSelectGroupType.url)
+        <a class="govuk-link_exceed_group" href="@routes.ManageGroupController.showManageGroups(None,None).url"> @* Redirect link to correct url*@
+            Manage access groups<span class="govuk-visually-hidden"> contact details</span>
```

Comment:
- use `<div class="govuk-button-group">` instead of a plain div, then you shouldn't need your "govuk-link_exceed_group" class (https://design-system.service.gov.uk/components/button/#grouping-buttons)
- Remove `<span class="govuk-visually-hidden"> contact details</span>`
- replace "Manage access groups" text with a message key